src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
```

Comment:
Daniel Zerbino <notifications@github.com> writes:
> Still, that doesn't make Biodata a good name. May I suggest Specimen, which
> describes both individuals and samples?

This is intended to contain the metadata for all of the
biological stuff that exists outside of the computer.  It will
end up with more objects than are currently there (like sequencing experiements).

I can see that Biodata maybe not the best name, however specimen
doesn't capture it either.